prow/plugins/label/label.go

```diff
@@ -42,8 +42,8 @@ type assignEvent struct {
 }
 
 var (
-	labelRegex              = regexp.MustCompile(`(?m)^/(area|priority|kind)\s*(.*)$`)
-	removeLabelRegex        = regexp.MustCompile(`(?m)^/remove-(area|priority|kind)\s*(.*)$`)
+	labelRegex              = regexp.MustCompile(`(?m)^/(area|priority|kind|sig)\s*(.*)$`)
```

Comment:
Technically seems like we can put this into one labelRegex with `^/(remove-)?(area|priority)` and check whether group(1) is "" or not

I realize this behavior pre-exists though so feel free to ignore (even if you agree)